README.md

```diff
@@ -103,6 +111,9 @@ The provided data-readers' tags are:
 - `#config/env` Tags a string, interpreted as the name of an environment variable, and yields the string value of the environment variable. If the environment does not have that entry, then the var will use its default value or remain unbound.
 - `#config/file` Tags a string, interpreted as a path to a file, and yields the string contents of the file. If the file does not exist, then the var will use its default value or remain unbound.
 - `#config/edn` Tags a string, interpreted as a single EDN-formatted object, and yields the read object.  When composed with `#config/env` or `#config/file`, if the external value is not provided, then the var will use its default value or remain unbound.
+- `#config/etcd` Tags a string, interpreted as a key to lookup a value in etcd.
+- `#config/lookup` Tags a string, looks up a config value from another key in
```

Comment:
I'm not sure I see the value in `#config/lookup`. This library should be focused on configuring values from raw data. Making larger composite values out of them can easily be done orthogonally to this library.